src/testHelpers/withTimeMachine.jsx

```diff
@@ -0,0 +1,25 @@
+import React, { useEffect } from 'react';
+import { element } from 'prop-types';
+import {
+  startTimeMachine,
+  resetTimeMachine,
+} from '../../.storybook/time-machine';
+// This affects the global Date object for the storybook application, to ensure consistency in chromaticQA testing.
+const WithTimeMachine = ({ children }) => {
+  // set time, but leave it ticking, so knobs work
+  startTimeMachine(false);
+  // setting tick to false, prevents the timemachine from ticking,
+  // so the timestamp displayed is frozen, and there are no side effects.
+  useEffect(() => {
+    resetTimeMachine();
+    startTimeMachine(true);
+    return () => {
```

Comment:
Wrapper needed?
```suggestion
return resetTimeMachine;
```

Reply:
Functionality is still there without it, what does the wrapper do/improve on in this case?

Reply:
Functionality is there without it, what does the wrapper do/improve on in this case?

Reply:
I'm suggesting removing the wrapper you have, not adding another one :)

before:
```
return () => {
  resetTimeMachine ();
}
```
after:
```
return resetTimeMachine;
```

Reply:
I don't even have the excuse that it's friday :P, nice catch, my b

Reply:
👍 my two-word comment wasn't great

https://www.youtube.com/watch?v=_K-L9uhsBLM